test/test_cpp_extensions_aot.py

```diff
@@ -21,10 +21,10 @@
 class TestCppExtensionAOT(common.TestCase):
     """Tests ahead-of-time cpp extensions
 
-    NOTE: run_test.py's test_cpp_extensions_aot_no_ninja target
-    also runs this test case, but with ninja disabled. If you are debugging
+    NOTE: run_test.py's test_cpp_extensions_aot_ninja target
+    also runs this test case, but with ninja enabled. If you are debugging
```

Comment:
Ditto, this should read either neutral or read as if "test_cpp_extensions_aot (with ninja)" is the default. Developers of features for cpp extensions should run test_cpp_extensions_aot (with ninja) because it is faster